File: cppProject/actuador.cpp
#include "actuador.h"

actuador::actuador():instrumentacion(){VariableManipulada = NewVariableManipulada;}

actuador::actuador(const std::string& NewId,const std::string& NewZonaId,bool NewEstado,
				   const std::string& NewVariableManipulada):instrumentacion(NewId,NewZonaId,NewEstado){
	VariableManipulada = NewVariableManipulada;
}

//obtener atributos
inline std::string actuador::getVariableManipulada()const{return sensor.VariableManipulada;}

//establecer atributos
inline void actuador::setVariableManipulada(const std::string& NewVariableManipulada){
	VariableManipulada = NewVariableManipulada;
}

File: pythonProject/actuador.py
class actuador():
	def __init__(self,Id,ZonaId,VariableManipulada,Estado=True):		
		self.Id = Id
		self.ZonaId = ZonaId
		self.VariableManipulada = VariableManipulada
		self.Estado = Estado		
	#obtener atributos
	def getId(self):
		return self.Id		
	def getZonaId(self):
		return self.ZonaId		
	def getVariableManipulada(self):
		return self.VariableManipulada		
	def getEstado(self):
		return self.Estado				
	#establecer atributos	
	def setId(self,NewId):
		self.Id = NewId
	def setZonaId(self,NewZonaId):
		self.ZonaId = NewZonaId
	def setVariableManipulada(self,NewVariableManipulada):
		self.VariableManipulada = NewVariableManipulada
	def setEstado(self):
		self.Estado = ~self.Estado
		